package.json

```diff
@@ -40,7 +40,7 @@
   },
   "peerDependencies": {
     "lodash": "^4.17.4",
-    "mobx-state-tree": "^1.0.1"
+    "mobx-state-tree": "^1.0.0"
```

Comment:
why ?

Reply:
Because I don't need the 1.0.1 to work, just an older version of 1.0.0